krikri.gemspec

```diff
@@ -22,25 +22,25 @@ Gem::Specification.new do |s|
   s.test_files = Dir["spec/**/*"]
 
   s.add_dependency "rails", "~> 4.1.6"
-  s.add_dependency "rails_config", "~>0.4.0"
-  s.add_dependency "audumbla", '~> 0.1'
+  s.add_dependency "rails_config", "~> 0.4.0"
+  s.add_dependency "audumbla", "~> 0.2"
   s.add_dependency "rdf-turtle", "~>1.1.8"
   s.add_dependency "rdf", "~> 1.1.13"
   s.add_dependency "dpla-map", "4.0.0.0.pre.13"
-  s.add_dependency "rest-client"
-  s.add_dependency "rdf-marmotta", '>= 0.0.6'
-  s.add_dependency "blacklight", "~>5.8.0"
-  s.add_dependency "therubyracer"
+  s.add_dependency "rest-client", "~> 2.0"
+  s.add_dependency "rdf-marmotta", ">= 0.0.6"
```

Comment:
Are we sure that `rdf-marmotta` won't have breaking changes between major versions?